sheep/gateway.c

```diff
@@ -128,7 +133,9 @@ static struct req_iter *prepare_erasure_requests(struct request *req, int *nr)
 	ctx = ec_init(ed, edp);
 	*nr = nr_to_send = (opcode == SD_OP_READ_OBJ) ? ed : edp;
 	strip_size = SD_EC_DATA_STRIPE_SIZE / ed;
-	reqs = xzalloc(sizeof(*reqs) * nr_to_send);
+	reqs = zalloc(sizeof(struct request) * nr_to_send);
```

Comment:
Sorry but sizeof operand must be struct **req_iter**, not struct request. I'll fix in the next PR.